src/items/held-item-manager.ts

```diff
@@ -50,7 +49,7 @@ export class HeldItemManager {
 
   generateSaveData(): HeldItemSaveData {
     const saveData: HeldItemSaveData = [];
-    for (const [id, item] of this.getHeldItemEntries()) {
+    for (const [id, item] of this.heldItems.entries()) {
```

Comment:
```suggestion
    for (const [id, item] of this.heldItems) {
```
Map iterator defaults to entries